--- mmd.py.orig	2018-05-04 21:50:05.841418500 +0200
+++ mmd.py.new	2018-05-04 22:53:26.526557900 +0200
@@ -12,6 +12,8 @@
 _MMD_LIB = None
 _LIB_LOCATION = None
 
+import sys
+
 def load_mmd():
     """Loads libMultiMarkdown for usage"""
     global _MMD_LIB
@@ -20,6 +22,8 @@
         lib_file = 'libMultiMarkdown' + SHLIB_EXT[platform.system()]
         _LIB_LOCATION = os.path.abspath(os.path.join(DEFAULT_LIBRARY_DIR, lib_file))
 
+        if getattr(sys, 'frozen', False):
+            _LIB_LOCATION = os.path.abspath(os.path.join(sys._MEIPASS, lib_file))
         if not os.path.isfile(_LIB_LOCATION):
             _LIB_LOCATION = ctypes.util.find_library('MultiMarkdown')
 
